src/App.js

```diff
@@ -20,6 +21,7 @@ class App extends Component {
       <Switch>
         <Route path='/component/drawer/:type' component={DrawerFramePage} />
         <Route path='/component/top-app-bar/:type' component={TopAppBarFramePage} />
+        <Route path='/component/iframe/:type' component={withRouter(props => <IFrameRoutes {...props} />)} />
```

Comment:
I'm pretty sure instead of `component={...`, you can just do
```
render={IframeRoutes}
```

Then in IframeRoutes component, you'd wrap withRouter(IFrameRoutes). Ping me to experiement with the correct syntax.